src/subscribables/observable.js

```diff
@@ -25,8 +25,13 @@ ko.observable = function (initialValue) {
 
     if (DEBUG) observable._latestValue = _latestValue;
     observable.peek = function() { return _latestValue };
-    observable.valueHasMutated = function () { observable["notifySubscribers"](_latestValue); }
-    observable.valueWillMutate = function () { observable["notifySubscribers"](_latestValue, "beforeChange"); }
+    observable.valueHasMutated = function () {
+        observable.updateVersion();
```

Comment:
I also thought about using the notification, but if the observable is rate-limited, that would change when a sleeping computed would see the new value. Also how would it know when a sleeping computed has changed?